content/spin/build.md

```diff
@@ -3,7 +3,7 @@ template = "spin_main"
 date = "2022-03-14T00:22:56Z"
 enable_shortcodes = true
 [extra]
-url = "https://github.com/fermyon/spin/blob/main/docs/content/build.md"
+url = "https://github.com/fermyon/developer/blob/main//content/spin/build.md"
```

Comment:
Double slash?

Reply:
good catch!

Reply:
That URL is just embedded in the `edit on github` button. The `extra.url` is used nowhere else.